docs/modules/ROOT/pages/extensibility.adoc

```diff
@@ -79,7 +79,7 @@ They can be deployed as-is or used as templates for customization.
 
 Some presets are:
 
-* https://github.com/OpenZeppelin/cairo-contracts/blob/ad399728e6fcd5956a4ed347fb5e8ee731d37ec4/src/openzeppelin/account/Account.cairo[Account]
+* https://github.com/OpenZeppelin/cairo-contracts/blob/main/src/openzeppelin/account/presets/Account.cairo[Account]
```

Comment:
we avoid linking to `main` directly to preven broken links down the road. We prefer permalinks, this is, linking to a specific commit hash.

Reply:
thanks! In that case, I've also changed the link below to permalink :)

Reply:
Good catch!